package/utils/px5g-wolfssl/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2020 Paul Spooren <mail@aparcar.org>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=px5g-wolfssl
```

Comment:
Is there any specific reason for complete new package and not adding this into existing `px5g` package?

Reply:
Imho it's nicer than combining this into one package `px5g`, as in that way we don't need to filter build-dependencies and yet won't require users of either package to build both mbedtls and wolfssl as build-dependencies of the combined `px5g` package. px5g-wolfssl anyway doesn't share any code with px5g-mbedtls.

Reply:
I agree with @dango, the codebase is entirely different